jekyll/_includes/snippets/language-guided-tour-cards.md

```diff
@@ -0,0 +1,37 @@
+{::options parse_block_html="true" toc_levels="3..6" /}
+
+<div id="cards">
+<div class="card">
+
+{% include snippets/language-card.md lang=include.lang %}
+
+<div id="completion-time" class="card">
+<img alt="Completion Time" src="{{ site.baseurl }}/assets/img/docs/duration.svg">
+<span>50 min read</span>
```

Comment:
replace w/ variable once final copy/mock is discussed w/ @danbeere

Reply:
also considering refactoring these 3 lines into an reusable include

Reply:
nit: Should we rename the directory? We usually use `snippets` for everything that has to do with code snippets and that might be confusing